scipy/stats/stats.py

```diff
@@ -4840,13 +5052,19 @@ def ks_2samp(data1, data2):
     that, like in the case of the one-sample K-S test, the distribution is
     assumed to be continuous.
 
-    This is the two-sided test, one-sided tests are not implemented.
-    The test uses the two-sided asymptotic Kolmogorov-Smirnov distribution.
+    For small enough sample sizes, the computation is exact.  For large sizes,
```

Comment:
Maybe add what small enough means

Reply:
I was deliberately vague here, as I didn't want to commit to any particular size in the API's documentation and be bound to that forever. Computational resources have changed greatly since the first version of the K-S testing code went into SciPy.
Currently I'm using `max(n1, n2) <= 10,000` to be small enough.
Suggestions for how to word this?

Reply:
i think you can write is explicitly, even if one decides to change the default value later, i don't see a problem to simply update the doc.
This is what the R documentation of `ks.test` says:
"Exact p-values are not available for the two-sample case if one-sided or in the presence of ties. If exact = NULL (the default), an exact p-value is computed if the sample size is less than 100 in the one-sample case and there are no ties, and if the product of the sample sizes is less than 10000 in the two-sample case."

Reply:
Changed.